### ISSUE TITLE
ImportError: cannot import name 'basestring' from 'pyparsing'

### ISSUE THREAD

**Issue by YarShev** [mindsdb/dfsql#41]
Hi guys, I noticed that `dfsql` doesn't work with new versions of `pyparsing` (since 3.0.0). This should probably be fixed.

**Comment by YarShev** [comment]
```python
import dfsql.utils

./miniconda3/lib/site-packages/dfsql/utils.py in <module>

----> 1 from pyparsing import basestring
      2
      3 from dfsql.engine import pd
      4 from dfsql.exceptions import QueryExecutionException
      5

ImportError: cannot import name 'basestring' from 'pyparsing' (./miniconda3/lib/site-packages/pyparsing/__init__.py)
```

**Comment by btseytlin** [comment]
Thanks.

This is fixed in https://github.com/mindsdb/dfsql/pull/39, but the mysterious Ray/Dask issues are preventing me from releasing the fix. You can checkout `basestring_fix` if you want to test dfsql without this problem

**Comment by YarShev** [comment]
Yes, I do not see the issue related to the import error in `basestring_fix` branch but it would be great to have a new release of `dfsql` to support it in Modin as we normally do.

**Comment by btseytlin** [comment]
> Yes, I do not see the issue related to the import error in `basestring_fix` branch but it would be great to have a new release of `dfsql` to support it in Modin as we normally do.

Absolutely! But currently dfsql doesn't pass tests on OSX with Dask engine and on Ubuntu with Ray engine. Can't make a release until this is resolved

**Comment by YarShev** [comment]
Hope this will happen soon.

**Comment by YarShev** [comment]
@btseytlin, what is the status on this issue?

**Comment by btseytlin** [comment]
> @btseytlin, what is the status on this issue?

CC @StpMax

**Comment by tomhuds** [comment]
@StpMax can we close?
